utils-intervalrdd/src/main/scala/org/bdgenomics/utils/interval/array/IntervalArray.scala

```diff
@@ -122,6 +129,15 @@ trait IntervalArray[K <: Interval[K], T] extends Serializable {
     }
   }
 
+  /**
+   * Recursively finds an index in array that overlaps with the query rr, if exists.
+   *
+   * @param rr the query Interval.
```

Comment:
the query Interval. &rarr; Query interval.

(should be complete sentences, or at least sentence fragments punctuated and capitalized as such)